configdefinitions/src/vespa/stor-distribution.def

```diff
@@ -29,13 +32,15 @@ ready_copies int default=0
 ##   - ready_copies == redundancy
 ##   - Only one level of hierarchical grouping defined.
 ##   - That level distributes copies to all defined groups.
+## TODO: seems unused
```

Comment:
This config is inferred by the model based on the topology of the cluster and is used by the distributor bucket activation logic. Removing it would require more in-depth analysis of its current usage to determine if doing so is safe or not.